lib/galaxy/config/sample/themes_conf.yml.sample

```diff
@@ -74,3 +74,20 @@ smoky:
       active: "#FF206E"
     logo:
       img: "/static/favicon.svg"
+
+anvil:
+  masthead:
+    color: "#012840"
+    height: "3rem"
+    text:
+      color: "#f6f7f4"
+      hover: "#e0dd10"
+      active: "#ffff"
+    link:
+      color: transparent
+      hover: transparent
+      active: "#00090e"
+    logo:
+      height: "1.8rem"
```

Comment:
Thanks for working on this. Overall this looks fine to me, but I would prefer `masthead-logo-height` to not be a theme variable, but rather derived as ratio of the masthead head height as you suggest above.

Reply:
Right now the `height` is optional and is only included in the sample theme until I can figure out how to get it to be recalculated.  I was hoping @ElectronicBlueberry's suggestion would fix that, but I'm still seeing the value for `--masthead-logo-height` to be `2.5 * 0.6` and not `3.0 * 0.6` as it should be if it was using the masthead height as defined in the theme.  Laila, do you have any suggestions?